protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -669,4 +708,18 @@ service VersioningService {
             }
         };
     }
+
+    rpc findRepositories(FindRepositories) returns (FindRepositories.Response){
+        option (google.api.http) = {
+            post: "/v1/versioning/workspaces/{workspace_name}/findRepositories"
+            body: "*"
+        };
+    };
+
+    rpc findRepositoriesBlobs(FindRepositoriesBlobs) returns (FindRepositoriesBlobs.Response){
+        option (google.api.http) = {
+            post: "/v1/versioning/workspaces/{workspace_name}/findRepositoriesBlobs"
+            body: "*"
+        };
+    };
```

Comment:
should we have to add
```
additional_bindings {
  post: "/v1/versioning/findRepositoriesBlobs"
  body: "*"
}
```
?

Reply:
this will search across workspaces?
None of our search are across workspace , backend. defaults to personal workspace , so I would say not required for now.

Reply:
we have added `workspace_name` in the request parameter like https://github.com/VertaAI/modeldb/pull/543/files#diff-e2ae718e9e813919505c1e78be8dc18bR425